Create transfer.portuguese.lang

TehPeGaSuS opened the pull request:
Portuguese translation for the transfer module!
Cheers!

Found by:
Patch by:
Fixes: 

One-line summary:


Additional description (if needed):


Test cases demonstrating functionality (if applicable):

TehPeGaSuS committed:
Create transfer.portuguese.lang

Portuguese translation for the transfer module!
Cheers!

TehPeGaSuS commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -0,0 +1,64 @@
+0xf30,Tempo de ligação esgotado em DCC: SEND %s (%s) a %lu/%lu
+0xf31,envio  (%lu)/%lu\n    Ficheiro: %s\n
+0xf32,envio esperou %lus\n    Ficheiro: %s\n
+0xf33,envio conn
```
not sure about what means "conn"

Cizzle replied:
It's the abbreviation for "connection".

TehPeGaSuS replied:
Fixed

TehPeGaSuS commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -0,0 +1,64 @@
+0xf35,NOTICE %s :Ligação imprópria (%s)\n
+0xf36,Ligação DCC imprópria: GET %s (%s!%s)
+0xf37,Iniciando DCC %ssend %s para %s
+0xf38,re
```
if the "re" is used in something like re-connect, then it's fine as it is

Cizzle replied:
It's  to be used in 0xf37 above it to make "resend" in some cases.

TehPeGaSuS replied:
Then, it's fine as it is

Cizzle replied:
But shouldn't "send" be "envio" or some form of it?

TehPeGaSuS replied:
I've completely missed that one.
Fixing it right now.

Cizzle commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -0,0 +1,64 @@
+0xf06,NOTICE %s :Erro de DCC socket; abortando ficheiros em espera.\n
+0xf07,Abortada obtenção dcc %s: Ficheiro vazio!
+0xf08,NOTICE %s :Ficheiro %s está vazio, abortando transferência.\n
+0xf09,  Send to  %s  Ficheiro\n
```
"Send to" needs translation?

TehPeGaSuS replied:
Fixed!

Cizzle commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -0,0 +1,64 @@
+0xf09,  Send to  %s  Ficheiro\n
+0xf0a,  ---------%s  --------------------\n
+0xf0b,  %s%s  %s  [ESPERANDO]\n
+0xf0c,  %s%s  %s  (%.1f%% cpmpletado)\n
```
"completado"?

TehPeGaSuS replied:
Fixed

Cizzle commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -0,0 +1,64 @@
+0xf1a,FALHADA tentativa de mover `%s' da localização temporária! Ficheiro perdido!
+0xf1b,Obrigado pelo ficheiro!\n
+0xf1c,NOTICE %s :Obrigado pelo ficheiro!\n
+0xf1d,Tranferência do ficheiro de utlizador de %s perdida; abortando.
```
"utilizador"?

TehPeGaSuS replied:
yep. userfile is "ficheiro de utilizador" in portuguese

Cizzle commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -0,0 +1,64 @@
+0xf20,Perdido envio dcc %s de %s!%s (%lu/%lu)
+0xf21,(!) pacote reget de %s para %s é inválido!
+0xf22,!! Retomando transferência do ficheiro atrás do ficheiro final desde %s para %s
+0xf23,!!! Tentando avançar na transferência de arquivos de utlizador
```
"utilizador"

TehPeGaSuS replied:
Fixed

Cizzle commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -0,0 +1,64 @@
+0xf23,!!! Tentando avançar na transferência de arquivos de utlizador
+0xf24,Retomando a transferência de ficheiros em %dk desde %s para %s
+0xf25,Transferência de ficheiro de utilizador concluída para %s.
+0xf26,terminado envio ddcc %s para %s
```
Start with capital T and "dcc".

TehPeGaSuS replied:
Fixed

Cizzle commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -0,0 +1,64 @@
+0xf29,NOTICE %s :Comprimento de ficheiro malformada.\n
+0xf2a,Ficheiro demasiado longo: cancelando envio dcc %s desde %s!%s
+0xf2b,Tempo de ligação esgotado na transferência do ficheiro de utilizador.
+0xf2c,Desligado %s (tempo de ligação esgotado na transferência do ficheiro)
```
... "de utilizador"?

TehPeGaSuS committed:
Update transfer.portuguese.lang

All rechecked. i didnt touched 0xf18 neither 0xf19 because i dont know with what replace it and i like it the way it is, because in portuguese the translation wouldnt make much sense.
Seems to be all completed now. and thanks for noticing the errors. :D

Cizzle commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -11,10 +11,10 @@
 0xf08,NOTICE %s :Ficheiro %s está vazio, abortando transferência.\n
-0xf09,  Send to  %s  Ficheiro\n
+0xf09,  Enviando para  %s  Ficheiro\n
 0xf0a,  ---------%s  --------------------\n
```
You'll want to add more "-" so the %s match up again with 0xf09.
And add more spaces at 0xf0b and 0xf0c so the third %s matches up with the %s at 0xf09.

TehPeGaSuS committed:
Update transfer.portuguese.lang

Check if everything's OK now.

TehPeGaSuS committed:
Update transfer.portuguese.lang

Fixed one last error.

Cizzle commented on src/mod/transfer.mod/language/transfer.portuguese.lang:
```diff
@@ -12,9 +12,9 @@
-0xf0b,  %s%s  %s  [ESPERANDO]\n
-0xf0c,  %s%s  %s  (%.1f%% completado)\n
+0xf0a,  ---------------%s  --------------------\n
+0xf0b,           %s%s  %s  [ESPERANDO]\n
```
The spaces are best added before the third %s.

TehPeGaSuS committed:
Update transfer.portuguese.lang

Fixed.

Cizzle commented:
LGTM

TehPeGaSuS commented:
Happy to contribute! :D
